TeensyDmx.h

```diff
@@ -56,6 +58,10 @@ struct RdmData
 static_assert((sizeof(RdmData) == 255),
               "Invalid size for RdmData struct, is it packed?");
 
+using rdmControllerCallback = void(*)(CallbackStatus, RdmData*);
```

Comment:
This is a type, so should have a capital R.

Reply:
Done